pkg/client/cmd/app.go

```diff
@@ -1091,6 +1092,38 @@ func appChangeTeam(cmd *cobra.Command, args []string) {
 	fmt.Println("Team changed successfully")
 }
 
+var appSetVHostsCmd = &cobra.Command{
+	Use:   "set-vhosts <name> [vhost, ...]",
+	Short: "Set vhosts for the app",
+	Long: `Set vhosts for the app on clusters with ingress integration.
+
+  $ teresa app set-vhosts myapp myapp.mydomain
+
+  You can also provide more than one vhost at a time:
+
+  $ teresa app set-vhosts myapp myapp.mydomain myapp.anotherdomain`,
+	Run: appSetVHosts,
+}
+
+func appSetVHosts(cmd *cobra.Command, args []string) {
+	if len(args) == 0 {
```

Comment:
hm, what about `if len(args) < 2`  because we need at least two args (app name and one vhost)

Reply:
It makes  sense.